### PR TITLE
Remove unneeded user info data

### PR DESCRIPTION
**Pull request by githinho**

### CODE REVIEW

**Comment by githinho** [comment]
@mj-ampnet @filipdujmusic please confirm we won't need this data

**Review by mightymatth — APPROVED** [review]

File: src/main/resources/db/migration/V10__trim_user_info.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+ALTER TABLE user_info
+    DROP COLUMN id_number,
+    DROP COLUMN date_of_birth,
+    DROP COLUMN place_of_birth,
+    DROP COLUMN document_type,
+    DROP COLUMN document_number,
+    DROP COLUMN document_country,
+    DROP COLUMN document_valid_from,
+    DROP COLUMN document_valid_until,
+    DROP COLUMN deactivated;
```
Comment (filipdujmusic) [inline src/main/resources/db/migration/V10__trim_user_info.sql:10]: not sure about this. Waiting for @Ivan Kardum to provide more info on the user data that we have to store. It looks like we will have to store following:
> First/Last Name
> Residence
> Date of birth
> Personal identification number

We'll decide today.

Comment (mj-ampnet) [reply src/main/resources/db/migration/V10__trim_user_info.sql:10]: Who is Ivan Kardum

**Comment by githinho** [comment]
Rejected because we need to have data forever. KYC provider stores data only for a year.

**Pull request closed without merging**